installers.json

```diff
@@ -0,0 +1,20 @@
+{
+  "installerURLs": {
+    "cdk" : {
+      "0" : "http://cdk-builds.usersys.redhat.com/builds/03-Mar-2016/cdk.zip", 
```

Comment:
Though the indices are arbitrary & meaningless, I've added some labels you might enjoy instead.